data/stateChange/stateChange.proto

```diff
@@ -28,8 +28,9 @@ message StateAccess {
 }
 
 message DataTrieChange {
-  ActionType  Type    = 1 [(gogoproto.jsontag) = "type"];
-  bytes       Key     = 2 [(gogoproto.jsontag) = "key"];
-  bytes       Val     = 3 [(gogoproto.jsontag) = "val"];
-  uint32      Version = 4 [(gogoproto.jsontag) = "version"];
+  ActionType  Type      = 1 [(gogoproto.jsontag) = "type"];
+  bytes       Key       = 2 [(gogoproto.jsontag) = "key"];
+  bytes       Val       = 3 [(gogoproto.jsontag) = "val"];
+  uint32      Version   = 4 [(gogoproto.jsontag) = "version"];
+  uint32      Operation = 5 [(gogoproto.jsontag) = "operation"];
 }
```

Comment:
remove empty line at the end of the file?

Reply:
actually the new line is missing

Reply:
Added empty line.